dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/dao/SimpleMongoDBStorageStrategy.java

```diff
@@ -53,8 +54,11 @@
     public static final String OBJECT_TYPE_VALUE = XMLSchema.ANYURI.stringValue();
     public static final String CONTEXT = "context";
     public static final String PREDICATE = "predicate";
-    public static final String OBJECT = "object";
+    public static final String PREDICATE_HASH = "predicate_hash";
+    public static final String OBJECT = "object_original";
```

Comment:
I'm pretty sure Mongo further condenses the data, so I'm not sure hashing is necessary in order for it to store in memory.  You're adding a lot of overhead to query.  I'm ok with adding it now if you think it's necessary.